adapters/repos/db/index.go

```diff
@@ -1901,13 +1905,19 @@ func (i *Index) findUUIDs(ctx context.Context,
 	for _, shardName := range shardNames {
 		var err error
 		var res []strfmt.UUID
-		if shard := i.localShard(shardName); shard != nil {
-			res, err = shard.FindUUIDs(ctx, filters)
-		} else {
-			res, err = i.remote.FindUUIDs(ctx, shardName, filters)
-		}
+		err = backoff.Retry(func() error {
+			if shard := i.localShard(shardName); shard != nil {
+				res, err = shard.FindUUIDs(ctx, filters)
+			} else {
+				res, err = i.remote.FindUUIDs(ctx, shardName, filters)
```

Comment:
`remote.FindUUIDs()` is making a call to a remote node. I don't think we need to retry here. Our focus is on managing eventual consistency on the host node

Reply:
we need to retry the localShard though

Reply:
We can isolate the localShard call in it's own retry and then as a follow up either call local `FindUUIDs` or call it on remote.

Reply:
yeah on it